src/o365/yammer/commands/group/group-user-remove.ts

```diff
@@ -0,0 +1,150 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import YammerCommand from "../../../base/YammerCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: number;
+  userId?: number;
+  confirm?: boolean;
+}
+
+class YammerGroupUserRemoveCommand extends YammerCommand {
+  constructor() {
+    super();
+  }
+
+  public get name(): string {
+    return `${commands.YAMMER_GROUP_USER_REMOVE}`;
+  }
+
+  public get description(): string {
+    return 'Removes a user from a Yammer group';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = args.options.id !== undefined;
+    telemetryProps.userId = args.options.userId !== undefined;
+    telemetryProps.confirm = (!(!args.options.confirm)).toString();
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const executeRemoveAction: () => void = (): void => {
+      let endpoint = `${this.resource}/v1/group_memberships.json`;
+
+      const requestOptions: any = {
+        url: endpoint,
+        headers: {
+          accept: 'application/json;odata.metadata=none',
+          'content-type': 'application/json;odata=nometadata'
+        },
+        json: true,
+        body: {
+          group_id: args.options.id,
+          user_id: args.options.userId
+        }
+      };
+
+      request
+            .delete(requestOptions)
+            .then((res: any): void => {
+              cb();
+            }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+    };
+
+    if (args.options.confirm) {
+      executeRemoveAction();
+    }
+    else {
+      let messagePrompt = `Are you sure you want to remove you from the Group ${args.options.id}?`;
+      if (args.options.userId)
+        messagePrompt = `Are you sure you want to remove the user ${args.options.userId} from the Group ${args.options.id}?`;
+
+      cmd.prompt({
+        type: 'confirm',
+        name: 'continue',
+        default: false,
+        message: messagePrompt,
+      }, (result: { continue: boolean }): void => {
+        if (!result.continue) {
+          cb();
+        }
+        else {
+          executeRemoveAction();
+        }
+      });
+    }
+  };
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '--id <id>',
+        description: 'The Group ID to process'
+      },
+      {
+        option: '--userId [id]',
```

Comment:
The value between brackets should match the name of the option, eg `--userId [userId]`